Sources/FrostflakeKit/FrostflakeIdentifier+Base58.swift

```diff
@@ -0,0 +1,48 @@
+extension FrostflakeIdentifier {
+    private static let _base58Alphabet = "123456789ABCDEFGHJKLMNPQRSTUVWXYZabcdefghijkmnopqrstuvwxyz"
+    private static let _base58Characters = Array(_base58Alphabet) // Convert to array for direct indexing
+    // ASCII value of 'z' is 122, so we create an array of size 123.
+    private static let _base58AlphabetIndexByChar: [Int?] = {
+        var indexes = [Int?](repeating: nil, count: 123) // 'z' is the highest ASCII character in the alphabet.
+        for (index, char) in _base58Alphabet.utf8.enumerated() {
+            indexes[Int(char)] = index
+        }
+        return indexes
+    }()
+
+    public var base58: String {
+        var number = self
+        var encodedChars: [Character] = [] // Use array to collect characters
+
+        while number > 0 {
+            let remainder = Int(number % 58)
```

Comment:
May be we should use some constant instead of hardcoded number here?

Reply:
Sure, done.